packages/purpose-process/src/utilities/errorMappers.ts

```diff
@@ -48,3 +49,16 @@ export const deletePurposeVersionErrorMapper = (
     .with("organizationIsNotTheConsumer", () => HTTP_STATUS_FORBIDDEN)
     .with("purposeVersionCannotBeDeleted", () => HTTP_STATUS_CONFLICT)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const rejectPurposeVersionErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with(
+      "purposeNotFound",
+      "purposeVersionNotFound",
+      () => HTTP_STATUS_NOT_FOUND
+    )
+    .with("organizationIsNotTheProducer", () => HTTP_STATUS_FORBIDDEN)
+    .with("notValidVersionState", () => HTTP_STATUS_BAD_REQUEST)
+    .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
```

Comment:
`missingRejectionReason` is missing, I think it should be a `BAD_REQUEST`

Reply:
Fixed, thanks!